pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -116,7 +116,7 @@ func New(address string, port uint, imageFsInfoProvider ImageFsInfoProvider, roo
 
 	if _, err := os.Stat(rootPath); err != nil {
 		if os.IsNotExist(err) {
-			if err := os.MkdirAll(path.Clean(rootPath), 0750); err != nil {
+			if err := os.MkdirAll(path.Clean(rootPath), 0755); err != nil {
```

Comment:
Could you please elaborate why changing to world read? Is it  because of the remapped user needs to read it? Might be a minor security issue as other users may read sensitive data (if any?) as well.

Reply:
The remapped user (non-root, non-root-group) needs to be able to read directories under it.

We could likely go with 0751 -- would that be acceptable?